llvm_mode/afl-llvm-pass.so.cc

```diff
@@ -104,16 +105,17 @@ bool AFLCoverage::runOnModule(Module &M) {
      __afl_prev_loc is thread-local. */
 
   GlobalVariable *AFLMapPtr =
```

Comment:
Let's change the naming to something else, since' it's not longer AFL's coverage map. How about CoverageMapPtr or BBCoverageMapPtr?